.github/workflows/basic-sanity-ubuntu-22.yml

```diff
@@ -14,7 +14,14 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
-
```

Comment:
Try adding "Permissions" [clause](https://github.com/loxilb-io/loxilb/blob/042ba71ebbfd6f2f9596a4708fd8c893b0cc60d0/.github/workflows/docker-image.yml#L19) to rectify docker push issue.